.gitignore

```diff
@@ -13,6 +13,13 @@ debug/
 release/
 testresults/
 
+# Cordova
+cordova/www/
+cordova/platforms/
+cordova/plugins/
+cordova/resources/
+cordova/package-lock.json
```

Comment:
I think that with `cordova/` is enough?

Reply:
I don't understand what you said. I don't think `cordova/` can replace what you quoted

Reply:
I think `cordova/` will ignore all the subfolders too, or at least it seemed in my local in a test that I did yesterday.

Reply:
Yes but it will ignore files inside `cordova/` as well ? If not, I will make the changes

Reply:
You are right, I forgot that now we have some files inside the `cordova` folder. I thought it was used only when generating.
Maybe there is a cleaner way to add it to `gitignore`, but I'm not aware of any. So is good as is.